book/src/main_event_loop.md

```diff
@@ -86,6 +86,12 @@ Filename: <a class=file-link href="https://github.com/gtk-rs/gtk4-rs/blob/master
 {{#rustdoc_include ../listings/main_event_loop/4/main.rs:callback}}
 ```
 
+> There are two async ecosystems in Rust: [`tokio`](https://docs.rs/tokio) and [`futures`](https://docs.rs/futures).
```

Comment:
That sentence doesn't make much sense to me. tokio is also based on the futures crate, just that tokio has its own `Stream` trait.

Maybe say that there are multiple async runtimes in Rust: e.g. tokio which is most widely used, async-std and smol. And that another async runtime is provided by the glib crate.

Also that some crates expect a specific runtime to be used and you would have to implement interop between the different runtimes, e.g. via channels. And in the case of tokio, smol and glib you also need to make sure there is an actual runtime running because it doesn't happen implicitly.

Reply:
also async-std does not have any (stable) stream support at all, for example